### ISSUE TITLE
Webpack support

### ISSUE THREAD

**Issue by alpox** [arabsight/aurelia-mdl-plugin#3]
I tried to integrate the plugin with webpack but didn't have much success yet. I keep running into this error:
![auswahl_010](https://cloud.githubusercontent.com/assets/12485194/18472978/b023b460-79ba-11e6-84db-0be3e25de594.png)

And the dist folder doesn't seem to have the structure as other aurelia apps. Maybe that is what causes the problem? I'll keep digging :-)

**Comment by alpox** [comment]
I found a workaround for this.
With adding:

```
"aurelia": {
    "build": {
      "resources": [
        "aurelia-mdl-plugin/mdl.js"
      ]
    }
  }
```

to the `package.json` file did the trick.

**Comment by arabsight** [comment]
@alpox, I never worked with webpack before, but I'll give a try. In the mean time if you found a solution feel free to submit a PR.

**Comment by arabsight** [comment]
@alpox its fixed (v0.1.6) no need to define resources in package.json
